GVRf/Framework/jni/objects/scene_object_jni.cpp

```diff
@@ -176,7 +179,7 @@ Java_org_gearvrf_NativeSceneObject_attachEyePointeeHolder(
         jlong jeye_pointee_holder) {
     SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
     EyePointeeHolder* eye_pointee_holder =
-            reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
+    reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
```

Comment:
Hi.
I just get this pull request and tried with the Ctrl+Shift+F. I find there is something different. 
Can you help go to Windows->Preference->C++->Code Style and check what your profile is?

Thanks in advance.